content/docs/start/data-and-model-versioning.md

```diff
@@ -82,35 +82,31 @@ outs:
     path: data.xml
 ```
 
-> \* See
-> [Large Dataset Optimization](/doc/user-guide/large-dataset-optimization) and
-> `dvc config cache` for more info. on file linking.
-
 </details>
 
 ## Storing and sharing
 
 You can upload DVC-tracked data or model files with `dvc push`, so they're
 safely stored [remotely](/doc/command-reference/remote). This also means they
 can be retrieved on other environments later with `dvc pull`. First, we need to
-setup a storage:
+setup a storage provider:
```

Comment:
Agree that "a storage" sounds incomplete. How about a "storage location" or a "[DVC remote](https://dvc.org/doc/command-reference/remote)" for terminology consistency? We use those terms a lot (not really "provider").

Reply:
changed